tst/edgeColoured.tst

```diff
@@ -0,0 +1,4 @@
+gap> g1 := NautyEdgeColoredGraph( [[[1,2],[3,10],[11,12],[4,9],[5,6],[7,8]],[[1,12],[2,3],[10,11],[4,5],[6,7],[8,9]],[[3,4],[9,10]]],12);;
+gap> g2 := NautyEdgeColoredGraph( [[[3,11],[9,10],[4,7],[1,2],[5,12],[6,8]],[[7,11],[4,10],[3,9],[1,5],[2,8],[6,12]],[[5,9],[10,12]]],12);;
+gap> IsomorphismGraphs(g1,g2);
+(1,2)(3,5)(4,9,10,12,8)(6,11)
```

Comment:
So we should also have a test here for two non-isomorphic graphs. And perhaps some corner/edge cases, like what happens if the two graphs have different numbers of vertices.

Reply:
(I can also help out with that, just don't want to forget it)